### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by c-mertes**
Bump DROP version and merge `dev` into `master`

### CODE REVIEW

File: drop/modules/mae-pipeline/MAE/filterSNVs.sh [hunk 1]
```diff
@@ -27,6 +27,7 @@ if [ $vcf_id != 'QC' ]; then sample_flag="-s ${vcf_id}"; fi
 $bcftools view  $vcf_file | \
     grep -vP '^##INFO=' | \
     awk -F'\t' 'BEGIN {OFS = FS} { if($1 ~ /^[^#]/){ $8 = "." }; print $0 }' | \
+    $bcftools norm -m-both | \
     $bcftools view ${sample_flag} -m2 -M2 -v snps -O z -o $tmp
 $bcftools index -t $tmp
```
Comment (nickhsmith) [inline drop/modules/mae-pipeline/MAE/filterSNVs.sh:32]: In the `separate_RVC` branch I've updated this to be more robust to multi-sample VCFs. Maybe we can use that version

**Review by nickhsmith — COMMENTED** [review]
For the MAE_filter. This works, but there are situations where a user has had VCFs of odd formats including VCFs with homozygous reference calls and multi-sample vcfs. In the `separate_RVC` branch there is an explicit selection to reduce corner-case errors.

**Review by nickhsmith — APPROVED** [review]

File: drop/template/Snakefile [hunk 2]
```diff
@@ -4,8 +4,6 @@ import drop
 import wbuild
 from wbuild.createIndex import ci
 
-configfile: "config.yaml"
-
```
Comment (mumichae) [inline drop/template/Snakefile:8]: Why was this removed? Do we want to change the default?

**Review by mumichae — APPROVED** [review]
A question on why we removed the config file from the Snakefile, but apart from that approved

**Review by vyepez88 — APPROVED** [review]

**Comment by nickhsmith** [comment]
This doesn't change the default. by default snakemake and wbuild looks for
a file called config.yaml, but if you do give another config using
--configfile, then it doesn't work correctly.

**Comment by mumichae** [comment]
I see, so as long as this doesn't change the default behaviour we don't need to change the docs
